pkg/controller/authentication/ingress.go

```diff
@@ -804,6 +810,13 @@ func platformOidcIngress(instance *operatorv1alpha1.Authentication, scheme *runt
 			},
 		},
 	}
+	if ingressUpdateRequired {
```

Comment:
@rashmi43 you can remove this whole block, we won't have a configmap kind of scenario here for the ingress.